sematic/ui/packages/main/src/pipelines/pod_lifecycle/PodLifecycle.tsx

```diff
@@ -0,0 +1,73 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Accordion from '@mui/material/Accordion';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import Box from '@mui/material/Box';
+import Chip from '@mui/material/Chip';
+import Typography from '@mui/material/Typography';
+import { usePipelinePanelsContext } from "src/hooks/pipelineHooks";
+import { useRunJobHistory } from 'src/hooks/podHistoryHooks';
+import { useRunPanelLoadingIndicator } from 'src/hooks/runDetailsHooks';
+import PodEventHistory from 'src/pipelines/pod_lifecycle/PodEventHistory';
+import styled from '@emotion/styled';
+import { theme } from '@sematic/common/lib/src/theme/mira';
+import CopyButton from '@sematic/common/lib/src/component/CopyButton';
+import { useCallback } from 'react';
+import Alert from '@mui/material/Alert';
+
+const StyledCode = styled.code`
+  font-size: 12px;
+  color: ${theme.palette.grey[500]};
+  cursor: text;
+  user-select: text;  
+`;
+
+// Separate component to assign key to the component, so that it will be re-mounted when runId changes
+function PodLifecycleWithRunId(prop: { runId: string }) {
+  const { runId } = prop;
+
+  const { value, loading } = useRunJobHistory(runId);
+
+  useRunPanelLoadingIndicator(loading);
+
+  const preventPropagation = useCallback((e: React.MouseEvent) => {
+    e.stopPropagation();
+  }, []);
+
+  if (!value) return null;
+  if (value?.length === 0) return <Typography variant="body2">No pod history found</Typography>;
+
+  return <>
+    <Alert severity="info">
+      The information in this tab is provided for debugging purposes.
+      The information it displays is periodically polled from Kubernetes,
+      and not all intermediate states may be represented in the timelines.
+    </Alert>
+    {value!.map(
+      (job, index) =>
+        <Accordion key={index} defaultExpanded={index === value.length - 1}>
+          <AccordionSummary expandIcon={<ExpandMoreIcon />} >
+            <Box>
+              <Chip label={job.kind} size="small" />
+              {' Kubernetes Job ID: '}
```

Comment:
Is this small `run` chip necessary?
<img width="112" alt="image" src="https://user-images.githubusercontent.com/1894533/231538131-f2a79ab3-5947-4b6a-ad61-c6cf963dbc8e.png">

Reply:
Thanks for asking. Do you want to take out the entire chip, or just not use the chip but still display a plain `job.kind` ?

Reply:
I believe it should be consistent with the rest of the rendering. For the Run id, we just have `ID: <run id>` in preformatted style, so maybe we should have `Job ID: <job id>` in preformatted style here as well.

<img width="165" alt="image" src="https://user-images.githubusercontent.com/1894533/231569424-727df62a-f529-4ee2-9a50-90ef9ba7f730.png">

Reply:
Sounds good. Made it that way. 

<img width="691" alt="image" src="https://user-images.githubusercontent.com/1046489/232152752-5d6fdb16-7357-467e-9f95-60d27c018102.png">